src/ray/raylet/scheduling/scheduling_ids.h

```diff
@@ -185,6 +185,17 @@ class ResourceID : public BaseSchedulingID<SchedulingIDTag::Resource> {
   /// "ClusterResourceSchedulerTest" have different namespaces.
   friend void SetUnitInstanceResourceIds(absl::flat_hash_set<ResourceID> ids);
 
+  bool IsPlacementGroupWildcardResource() const {
+    auto resource_name = this->Binary();
```

Comment:
auto resource_name = this->Binary();

std::string_view resource_name_view(resource_name);
std::string_view pattern("_group_");

// The length of {placement_group_id} is fixed, so we just need to checking that if the length and the pos of `_group_` is match.
if (resource_name_view.size() <= pattern.size() + 2 * PlacementGroupID::Size()) {
	return false;
}

auto idx = resource_name_view.size() - (pattern.size() + 2 * PlacementGroupID::Size());
return resource_name_view. substr(idx, pattern.size()) == pattern;